compiler/rustc_codegen_llvm/src/debuginfo/mod.rs

```diff
@@ -206,6 +206,10 @@ impl<'ll> DebugInfoBuilderMethods for Builder<'_, 'll, '_> {
         }
     }
 
+    fn get_dbg_loc(&self) -> Option<&'ll DILocation> {
+        Some(unsafe { llvm::LLVMGetCurrentDebugLocation2(self.llbuilder) })
```

Comment:
Is there any case that LLVMGetCurrentDebugLocation2 would return a value that would be translated to None?

Reply:
Yes! In case we e.g. have `-Z debuginfo=0`.

Reply:
It looks great! Thank you!